### PR TITLE
Vovagusev

### PR DESCRIPTION
**Pull request by Invalid-coder**
<h1>Description</h1>

*Created some models with the highest 98.5% accuracy*

<h2>How has this been tested?</h2>

- [x] Executed code few times from start to end without crashing
- [x] Only ran once
- [x] Code have dedicated unit tests

### CODE REVIEW

File: vovagusev_code/task_3/face_mask_detection.py [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+
+retVal = model.fit(train_data, validation_data=test_data, epochs=15)
+
+"""**Let's visualize the result of training.**"""
+
+plt.plot(retVal.history['loss'], label = 'training loss')
+plt.plot(retVal.history['accuracy'], label = 'training accuracy')
+plt.legend()
+
+"""Is it bad when the learning curve looks almost like a line?
```
Comment (Yullia) [inline vovagusev_code/task_3/face_mask_detection.py:120]: You compare loss and accuracy at the same plot, that's why it looks a little weird.
For better visualization you could use separate plots for loss and accuracy:
plt.plot(retVal.history['accuracy'], label='train accuracy')
plt.plot(retVal.history['val_accuracy'], label='val accuracy')
plt.legend()

**Review by meanalexrin — CHANGES_REQUESTED** [review]
Please remove `facenet_keras.h5` from git :D
Storing models on the git itself is a bad tone
Just leave comments on where it should be downloaded 

Other than that - LGTM
